corokafka/utils/corokafka_offset_manager.cpp

```diff
@@ -21,7 +21,14 @@ namespace Bloomberg {
 namespace corokafka {
 
 OffsetManager::OffsetManager(corokafka::ConsumerManager& consumerManager) :
-    _consumerManager(consumerManager)
+    OffsetManager(consumerManager, std::chrono::milliseconds(-2))
```

Comment:
Could we perhaps use 0 here instead of -2, and change the logic in queryOffsetsFromBroker() to be > 0 instead of >= 0? It seems wrong to configure a 0-ms timeout.